frontend/src/constants/localStorage.ts

```diff
@@ -28,3 +28,8 @@ export const FEATURE_TOUR_KEY_PREFIX = 'has-seen-feature-tour-'
  * Key to store whether a user has seen the emergency contact number modal in localStorage.
  */
 export const EMERGENCY_CONTACT_KEY_PREFIX = 'has-seen-emergency-contact'
+
+/**
+ * Key to store when was the last time user has seen the admin feedback modal
+ */
+export const ADMIN_FEEDBACK_HISTORY_PREFIX = 'last-seen-admin-feedback'
```

Comment:
Oic, you decided to use local storage instead of db for this instead? Is there a reason why? Or is it just because its easier?

Reply:
Yep, just cause its faster, cause I realised we don't really have a patch to endpoint to our user details. If I go withe DB method will have to 
1. Change user schema
2. Create endpoint to patch user doc

Which can be quite extensive and likely increase dev time.

Have noted this in the PR description and will KIV on whether to 'hard set' the frequency to the user schema